pkg/i2gw/providers/istio/converter.go

```diff
@@ -302,7 +318,11 @@ func (c *converter) convertGateway(gw *istioclientv1beta1.Gateway, fieldPath *fi
 			GatewayClassName: K8SGatewayClassName,
 			Listeners:        listeners,
 		},
-	}, nil
+	}
+
+	notify(notifications.InfoNotification, fmt.Sprintf("successfully converted to Kubernetes Gateway \"%v/%v\"", gateway.Namespace, gateway.Name), gw)
```

Comment:
```suggestion
	notify(notifications.InfoNotification, fmt.Sprintf("successfully converted to Gateway \"%v/%v\"", gateway.Namespace, gateway.Name), gw)
```

I'd avoid using the `kubernetes` word here.

Reply:
Why? Kubernetes Gateway is useful to indicate that this is in opposed to "Istio Gateway"

Reply:
Because the information on the Istio Gateway was already embedded in the gw object. If we display the Kind of such an object, it's clear the source is an Istio Gateway, and should be clear the output is a Gateway API Gateway, without explicitly stating it. But yeah it's fine